src/main/java/com/iemr/hwc/controller/wo/UserBiomectrics.java

```diff
@@ -0,0 +1,60 @@
+package com.iemr.hwc.controller.wo;
+
+import com.iemr.hwc.controller.common.master.CommonMasterController;
+import com.iemr.hwc.data.registrar.FingerPrintDTO;
+import com.iemr.hwc.service.location.LocationServiceImpl;
+import com.iemr.hwc.service.registrar.RegistrarServiceImpl;
+import com.iemr.hwc.utils.response.OutputResponse;
+import io.swagger.annotations.ApiOperation;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping(value = "wo/user_biometrics")
+public class UserBiomectrics {
+    //    private OutputResponse response;
+    private Logger logger = LoggerFactory.getLogger(CommonMasterController.class);
+
+    private LocationServiceImpl locationServiceImpl;
+
+    @Autowired
+    public void setLocationServiceImpl(LocationServiceImpl locationServiceImpl) {
+        this.locationServiceImpl = locationServiceImpl;
+    }
+
```

Comment:
An API to get fingerprints for a user is also required.

Reply:
New API implemented